apple-pie/styles/style.css

```
@@ -0,0 +1,33 @@
+
+
+.encabezado {
+    color: white;
+    font-size: 35px;
+    padding: 500px;
+    background-image: url(/apple-pie/images/apple-pie.jpg);
```

```suggestion
    background-image: url("../images/apple-pie.jpg");
```

Lo de las comillas es puramente opcional pero a mi me gusta jajaja soy un pijolis, ahora bien te estaba fallando porque no le estabas especificando correctamente la ruta, unica y exclusivamente, pero vamos que el resto genial!